pkg/api/store/workload/workload_store.go

```diff
@@ -410,3 +433,82 @@ func imageUpdated(apiContext *types.APIContext, data map[string]interface{}) (bo
 	}
 	return false, nil
 }
+
+func getSchedulingRule(rule string) (string, string, string) {
+	parts := strings.Split(rule, " ")
+	if len(parts) < 3 {
+		return "", "", ""
+	}
+	var key, operator, value string
+	key = parts[0]
+	i := 1
+	operator = parts[i]
+	i++
+	if operator == "not" {
```

Comment:
We don't have the `not` operator, but a [notin](https://github.com/rancher/types/blob/master/mapper/scheduling.go#L175)